src/applications/personalization/dashboard/components/claims-and-appeals-v2/ClaimsAndAppealsV2.jsx

```diff
@@ -183,9 +198,11 @@ ClaimsAndAppealsV2.propTypes = {
   hasAPIError: PropTypes.bool.isRequired,
   loadAppeals: PropTypes.bool.isRequired,
   loadClaims: PropTypes.bool.isRequired,
+  loadLighthouseClaims: PropTypes.bool.isRequired,
```

Comment:
I believe these prop types should be functions instead of booleans

Reply:
When I ran your branch locally, I got the warning about the prop types (same with the loadClaims prop):

![Screenshot 2023-06-29 at 4 53 02 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/8332986/bf4d3a28-6fe7-443d-8d5b-e4b6d4f4bed2)

Those props are coming from `mapDispatchToProps` which are action functions:

``` js
const mapDispatchToProps = {
  loadAppeals: getAppealsAction,
  loadClaims: getClaimsAction,
  loadLighthouseClaims: getLighthouseClaimsAction,
};
```

So im fairly certain those three prop types should be `PropTypes.func.isRequired`